test/optimizer/stats_calculator_test.cpp

```diff
@@ -0,0 +1,557 @@
+#include "optimizer/statistics/stats_calculator.h"
+
+#include "gtest/gtest.h"
+#include "optimizer/logical_operators.h"
+#include "optimizer/optimizer_context.h"
+#include "parser/expression/column_value_expression.h"
+#include "parser/expression/comparison_expression.h"
+#include "parser/expression/conjunction_expression.h"
+#include "parser/expression/constant_value_expression.h"
+#include "parser/expression/operator_expression.h"
+#include "test_util/end_to_end_test.h"
+#include "test_util/test_harness.h"
+
+namespace noisepage::optimizer {
+class StatsCalculatorTests : public test::EndToEndTest {
```

Comment:
Can you add a test with multiple parameters as well?

Reply:
Done